packages/react-codemirror/src/e2e_tests/syntax-validation.spec.tsx

```diff
@@ -16,6 +16,7 @@ test('Prop lint set to false disables syntax validation', async ({
   });
 });
 
+// TODO Should this be unskipped
```

Comment:
do you want me to look into this? Or is it relevant to the PR somehow?

Reply:
I've uncommented the test and it works, not sure if this was an oversight and we forgot to unskip it in the past